backtrace-library/src/main/java/backtraceio/library/services/BacktraceMetrics.java

```diff
@@ -182,17 +182,24 @@ public void enable(BacktraceMetricsSettings settings, String uniqueEventName) {
             throw new IllegalArgumentException("Unique event name must be defined!");
         }
 
-        this.applicationName = ApplicationHelper.getApplicationName(this.getContext());
-        this.applicationVersion = ApplicationHelper.getApplicationVersion(this.getContext());
+        ApplicationMetadataCache applicationMetadata = ApplicationMetadataCache.getInstance(this.getContext());
+        this.applicationName = applicationMetadata.getApplicationName();
+        this.applicationVersion = applicationMetadata.getApplicationVersion();
+        setStartupUniqueEventName(uniqueEventName);
```

Comment:
should this be called twice (here and line 191)?

Reply:
this is a bug. Fixed